packages/react/src/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap

```diff
@@ -2319,7 +2323,7 @@ exports[`DataTable should render 1`] = `
                             xmlns="http://www.w3.org/2000/svg"
                           >
                             <svg
-                              aria-hidden={true}
+                              aria-label="Clear search query"
```

Comment:
@dakahn Shouldn't this stay `aria-hidden={true}` here for the icon? 🤔

Reply:
If the SVG purely decorative, but I think in this case, it's a button, right?

Reply:
@dakahn for sure but wouldn't this be redundant if parent button element already has the same `aria-label`? 🤔

Reply:
Oh, you're right! Didn't catch that with the snippet above. My bad.